int32 allied_type
---
float32 intensity_controller_target
int32 error